src/_codux/boards/calendar-date/calendar-date.board.module.scss

```diff
@@ -0,0 +1,6 @@
+.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```

Comment:
Saw that in the other board you were using % for the width, doesn't really matter which one you choose here but I would either change this to % or the other one to viewport units for consistency.
Also if you choose viewport units, consider using `dvh` and `dvw`.